flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/schema/TableSourceTable.scala

```diff
@@ -44,8 +43,8 @@ import org.apache.flink.table.types.logical.RowType
  * @param tableSource The [[DynamicTableSource]] for which is converted to a Calcite Table
  * @param isStreamingMode A flag that tells if the current table is in stream mode
  * @param catalogTable Resolved catalog table where this table source table comes from
- * @param flinkContext The flink context
- * @param abilitySpecs The abilitySpec applied to the source
+ * @param flinkContext The flink context abilitySpecs use to generate corresponding digests
```

Comment:
@param flinkContext which is used to generate extra digests based on abilitySpecs ?